Modules/ConfigOptions.lua

```diff
@@ -502,6 +502,7 @@ return {
 		modList:NewMod("Multiplier:NearbyRareOrUniqueEnemies", "BASE", val, "Config", { type = "Condition", var = "Combat" })
 		modList:NewMod("Multiplier:NearbyEnemies", "BASE", val, "Config", { type = "Condition", var = "Combat" })
 		modList:NewMod("Condition:OnlyOneNearbyRareOrUniqueEnemy", "FLAG", val == 1, "Config", { type = "Condition", var = "Combat" })
+		enemyModList:NewMod("Condition:NearbyRareOrUniqueEnemy", "FLAG", val >= 1, "Config", { type = "Condition", var = "Combat" })
```

Comment:
There is an existing condition called "RareOrUnique", which is what crafting bench mods look for and which is set to true if you set the enemy to a boss, but I used a new condition for this because the RareOrUnique condition is also used by things like curses and I wanted these two states to be decoupled.